src/parser/content-parser.php

```diff
@@ -256,13 +262,26 @@ protected function source_block( $block, $registered_blocks, $filter_options ) {
 		/**
 		 * Filters a block when parsing is complete.
 		 *
+		 * @deprecated version 1.1.0
+		 * 
 		 * @param array $sourced_block An associative array of parsed block data with keys 'name' and 'attribute'.
 		 * @param string $block_name Name of the parsed block, e.g. 'core/paragraph'.
 		 * @param int $post_id Post ID associated with the parsed block.
 		 * @param array $block Result of parse_blocks() for this block. Contains 'blockName', 'attrs', 'innerHTML', and 'innerBlocks' keys.
 		 */
 		$sourced_block = apply_filters( 'vip_block_data_api__sourced_block_result', $sourced_block, $block_name, $this->post_id, $block );
 
+		/**
+		 * Filters a block when parsing is complete.
+		 *
+		 * @param array $sourced_block An associative array of parsed block data with keys 'name' and 'attribute'.
+		 * @param string $block_name Name of the parsed block, e.g. 'core/paragraph'.
+		 * @param int $post_id Post ID associated with the parsed block.
+		 * @param array $block Result of parse_blocks() for this block. Contains 'blockName', 'attrs', 'innerHTML', and 'innerBlocks' keys.
+		 * @param array $filter_options Options to filter using, if any.
+		 */
+		$sourced_block = apply_filters( 'vip_block_data_api__sourced_block_result_transform', $sourced_block, $block_name, $this->post_id, $block, $filter_options );
```

Comment:
Made a new filter because the previous one had an extra param added to it, so it's a breaking change. This avoids that.

Reply:
Because this is additive, I don't think this is a breaking update. Implementers of the filter pass an explicit number of accepted parameters, like `5` in your usage here:

https://github.com/Automattic/vip-block-data-api/blob/778509d781453d693e368a19e6ac71f52c99e186/src/graphql/graphql-api.php#L28

Prior users of the filter would register for up to `4` arguments, which would continue to work here as the newest parameter is added to the end. [The docs page for `add_filter()`](https://developer.wordpress.org/reference/functions/add_filter/) sort of touches on that, but you can test that using `4` (or any lower positive number) instead of `5` continues to work without a break.

Reply:
Fair point, I've made that change. It's down to just the existing filter being modified 👍🏾